Build issue on Windows

```
   Compiling inlyne v0.4.1 (C:\Users\ךינשגכהד\.cargo\git\checkouts\inlyne-73285eb0a8bf9611\6696c92)
error[E0432]: unresolved import `winit::platform::wayland`
  --> src/main.rs:72:22
   |
72 | use winit::platform::wayland::WindowBuilderExtWayland;
   |                      ^^^^^^^ could not find `wayland` in `platform`
   |
note: found an item that was configured out
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winit-0.28.7\src\platform\mod.rs:27:9
   |
27 | pub mod wayland;
   |         ^^^^^^^

warning: unexpected `cfg` condition name: `inlyne_tcp_metrics`
  --> src\opts\config.rs:81:11
   |
81 |     #[cfg(inlyne_tcp_metrics)]
   |           ^^^^^^^^^^^^^^^^^^
   |
   = help: expected names are: `clippy`, `debug_assertions`, `doc`, `docsrs`, `doctest`, `feature`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `ub_checks`, `unix`, and `windows`
   = help: consider using a Cargo feature instead
   = help: or consider adding in `Cargo.toml` the `check-cfg` lint config for the lint:
            [lints.rust]
            unexpected_cfgs = { level = "warn", check-cfg = ['cfg(inlyne_tcp_metrics)'] }
   = help: or consider adding `println!("cargo::rustc-check-cfg=cfg(inlyne_tcp_metrics)");` to the top of the `build.rs`
   = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
   = note: `#[warn(unexpected_cfgs)]` on by default

warning: unexpected `cfg` condition name: `inlyne_tcp_metrics`
   --> src/main.rs:789:27
    |
789 |                     #[cfg(inlyne_tcp_metrics)]
    |                           ^^^^^^^^^^^^^^^^^^
    |
    = help: consider using a Cargo feature instead
    = help: or consider adding in `Cargo.toml` the `check-cfg` lint config for the lint:
             [lints.rust]
             unexpected_cfgs = { level = "warn", check-cfg = ['cfg(inlyne_tcp_metrics)'] }
    = help: or consider adding `println!("cargo::rustc-check-cfg=cfg(inlyne_tcp_metrics)");` to the top of the `build.rs`
    = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration

error[E0599]: no method named `with_name` found for struct `WindowBuilder` in the current scope
   --> src/main.rs:178:25
    |
178 |                 wb = wb.with_name("inlyne", "")
    |                         ^^^^^^^^^
    |
help: there is a method `with_theme` with a similar name, but with different arguments
   --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winit-0.28.7\src\window.rs:382:5
    |
382 |     pub fn with_theme(mut self, theme: Option<Theme>) -> Self {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
warning: `inlyne` (bin "inlyne") generated 2 warnings
error: could not compile `inlyne` (bin "inlyne") due to 2 previous errors; 2 warnings emitted
error: failed to compile `inlyne v0.4.1 (https://github.com/Inlyne-Project/inlyne#6696c929)`, intermediate artifacts can be found at `C:\Users\0BAC~1\AppData\Local\Temp\cargo-installlQOrFg`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -32,8 +32,14 @@ jobs:
         with:
           prefix-key: 'v0-rust-2'
 
-      - name: Run test suite
+        # `wayland` requires an extra dependecy that's not installed by default
+        # on the runner. Avoid using that feature for ubuntu CI
+      - name: Run test suite (ubuntu)
+        if: matrix.os == 'ubuntu-latest'
         run: cargo test --workspace --no-default-features --features x11
+      - name: Run test suite (non-ubuntu)
+        if: matrix.os != 'ubuntu-latest'
+        run: cargo test --workspace
 
   lint:
     runs-on: ubuntu-latest

--- a/src/main.rs
+++ b/src/main.rs
@@ -68,9 +68,6 @@ use winit::event::{
 use winit::event_loop::{ControlFlow, EventLoop, EventLoopBuilder};
 use winit::window::{CursorIcon, Window, WindowBuilder};
 
-#[cfg(feature = "wayland")]
-use winit::platform::wayland::WindowBuilderExtWayland;
-
 pub enum InlyneEvent {
     LoadedImage(String, Arc<Mutex<Option<ImageData>>>),
     FileReload,
@@ -168,15 +165,16 @@ impl Inlyne {
             let mut wb = WindowBuilder::new().with_title(utils::format_title(&file_path));
 
             if let Some(ref pos) = opts.position {
-                wb = wb.with_position(winit::dpi::PhysicalPosition::new(pos.x, pos.y))
+                wb = wb.with_position(winit::dpi::PhysicalPosition::new(pos.x, pos.y));
             }
             if let Some(ref size) = opts.size {
-                wb = wb.with_inner_size(winit::dpi::PhysicalSize::new(size.width, size.height))
+                wb = wb.with_inner_size(winit::dpi::PhysicalSize::new(size.width, size.height));
             }
-            #[cfg(feature = "wayland")]
+            #[cfg(all(feature = "wayland", not(any(target_os = "macos", windows))))]
             {
-                wb = wb.with_name("inlyne", "")
-            };
+                use winit::platform::wayland::WindowBuilderExtWayland;
+                wb = wb.with_name("inlyne", "");
+            }
 
             Arc::new(wb.build(&event_loop).unwrap())
         };

